src/app/modules/shared/shared.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CookingTimePipe } from './pipes/cooking-time.pipe';
+
+
+
+@NgModule({
+  declarations: [ CookingTimePipe],
+  imports: [
+    CommonModule
+  ],
+  exports: [CookingTimePipe]
+})
+export class SharedModule { }
```

Comment:
Please create separate modules for this `pipe` 
```suggestion
export class CookingTimePipeModule { }
```
or move it to 
recipeList module